lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -28,6 +30,15 @@
                 render: function() {
                     var filledTemplate = this.tpl(this.model.toJSON());
                     HtmlUtils.setHtml(this.$el, filledTemplate);
+                    this.postRender();
+                },
+
+                postRender: function(){
+                    new CourseEnrollView({        /*jshint -W031 */
```

Comment:
I think instead of commenting out the error you should fix it by assigning a variable, e.g. ```this.enrollView = new CourseEnrollView({...```